<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>180</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Magnetic Field Control Window</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/nSOTScannerResources.qrc">
    <normaloff>:/nSOTScanner/Pictures/jj_schema.svg</normaloff>:/nSOTScanner/Pictures/jj_schema.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralwidget{
background-color: rgb(0, 0, 0);
}


QLabel{
color:rgb(168,168,168)
}

QLineEdit{
border: 1px solid rgb(100,100,100);
background-color:rgb(230,230,230);
}

QTextEdit{
border: 1px solid black;
background-color: black;
color:rgb(168,168,168);
}

QRadioButton{
color:rgb(168,168,168)
}</string>
   </property>
   <widget class="QPushButton" name="push_Servers">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#push_Servers{
background: rgb(144, 140, 9);
 border-radius: 4px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_display">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Field (T):</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>500</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_setpoint">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>115</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ramprate">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>145</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>115</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Setpoint (T)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>145</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ramp rate (T/min)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="push_persistSwitch">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Switch only works if the charging and persistant fields are the same. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#push_persistSwitch{
background: rgb(161, 0, 0);
 border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_switchStatus">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Persist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_GotoSet">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#push_GotoSet{
color: rgb(168,168,168);
background-color:rgb(0,0,0);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}

QPushButton:pressed#push_GotoSet{
color: rgb(168,168,168);
background-color:rgb(95,107,166);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>GOTO
SET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_GotoZero">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#push_GotoZero{
color: rgb(168,168,168);
background-color:rgb(0,0,0);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}

QPushButton:pressed#push_GotoZero{
color: rgb(168,168,168);
background-color:rgb(95,107,166);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>GOTO
ZERO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_viewField">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#push_viewField{
color: rgb(168,168,168);
background-color:rgb(0,0,0);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}

QPushButton:pressed#push_viewField{
color: rgb(168,168,168);
background-color:rgb(95,107,166);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>FIELD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_viewCurr">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#push_viewCurr{
color: rgb(168,168,168);
background-color:rgb(0,0,0);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}

QPushButton:pressed#push_viewCurr{
color: rgb(168,168,168);
background-color:rgb(95,107,166);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>CURR.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_viewVolts">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#push_viewVolts{
color: rgb(168,168,168);
background-color:rgb(0,0,0);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}

QPushButton:pressed#push_viewVolts{
color: rgb(168,168,168);
background-color:rgb(95,107,166);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>VOLTS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_hold">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#push_hold{
color: rgb(168,168,168);
background-color:rgb(0,0,0);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}

QPushButton:pressed#push_hold{
color: rgb(168,168,168);
background-color:rgb(95,107,166);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>HOLD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_clamp">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#push_clamp{
color: rgb(168,168,168);
background-color:rgb(0,0,0);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}

QPushButton:pressed#push_clamp{
color: rgb(168,168,168);
background-color:rgb(95,107,166);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>CLAMP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_toggleView">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#push_toggleView{
color: rgb(168,168,168);
background-color:rgb(0,0,0);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}

QPushButton:pressed#push_toggleView{
color: rgb(168,168,168);
background-color:rgb(95,107,166);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>Monitoring Charging</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fieldval">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>151</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Resources/nSOTScannerResources.qrc"/>
 </resources>
 <connections/>
</ui>
